lib/ASTGen/Sources/ASTGen/ASTGen.swift

```diff
@@ -38,14 +38,14 @@ enum ASTNode {
     }
   }
 
-  func bridged() -> ASTNodeBridged {
+  func bridged() -> BridgedASTNode {
```

Comment:
Sorry to be a bit pedantic here, but should this be a computed property? More generally: in idiomatic Swift, when should something be a computed property vs a function without arguments?

Reply:
> in idiomatic Swift, when should something be a computed property vs a function without arguments? 

My general impression is that a function cautions that the underlying computation is non-trivial performance-wise (e.g. `String.uppercased()`).

Reply:
I guess this is what I get for calling the PR "cleanup" rather than "use wrapper types" or something :P. I can change it, though TBH this is all likely to change a fair bit anyway so I'm not sure if it'll stick around.